script/import_scripts/vbulletin.rb

```diff
@@ -469,15 +473,15 @@ def close_topics
     sql = <<-SQL
       WITH closed_topic_ids AS (
         SELECT t.id AS topic_id
-        FROM #{TABLE_PREFIX}post_custom_fields pcf
```

Comment:
This query is for postgresql database (Discourse database) not for mysql (VB4 database).